scripts/local-static.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+name="./scripts/$(basename $0)"
+if test ! -f "$name"; then
+  echo "ERROR: run from root holochain directory: '$name'"
+  exit 127
+fi
+
+cargo fmt --check && cargo clippy -- $(xargs <.config/clippy-args.nix)
```

Comment:
just FYI, this would be the equivalent with the new scripts: 

first `nix develop .#coreDev` and then `script-holochain-tests-static-fmt && script-holochain-tests-static-clippy`

or in one command: `nix develop .#coreDev --command bash -c "script-holochain-tests-static-fmt && script-holochain-tests-static-clippy"`